tv_apps/remote-control/js/config.js

```diff
@@ -0,0 +1,139 @@
+/* global SettingsListener, evt */
+'use strict';
+
+(function(exports) {
+  var PREFIX = 'remote-control.';
+  var SETTINGS = {
+    'enabled': false,
+    'pairing-required': false,
+    'authorized-devices': null,
+    'server-ip': ''
+  };
```

Comment:
By reading more codes, I feel this concatenation may increase the effort of debugging and tracing code. Strongly suggest we always use complete name of keys.